packages/teams-js/test/public/search.spec.ts

```diff
@@ -0,0 +1,264 @@
+import { errorLibraryNotInitialized } from '../../src/internal/constants';
+import { GlobalVars } from '../../src/internal/globalVars';
+import { app } from '../../src/public/app';
+import { FrameContexts } from '../../src/public/constants';
+import { M365ContentAction } from '../../src/public/interfaces';
+import { _minRuntimeConfigToUninitialize } from '../../src/public/runtime';
+import { search } from '../../src/public/search';
+import { Utils } from '../utils';
+
+/* eslint-disable */
+/* As part of enabling eslint on test files, we need to disable eslint checking on the specific files with
+   large numbers of errors. Then, over time, we can fix the errors and reenable eslint on a per file basis. */
+
+const dataError = 'Something went wrong...';
+
+/**
+ * Type guard to determine if an action item is of M365Content Type
+ */
+function isM365ContentType(actionItem: unknown): actionItem is M365ContentAction {
+  // eslint-disable-next-line no-prototype-builtins
+  return actionItem && Object.prototype.hasOwnProperty.call(actionItem, 'secondaryId');
+}
+
+describe('Testing search capability', () => {
+  const mockErrorMessage = 'Something went wrong...';
```

Comment:
This const is the same value as the dataError const above. Probably you only need one?